### ISSUE TITLE
Fix slight misalignment with Select menu

### ISSUE THREAD

**Issue by jamesfan** [Workday/canvas-kit#728]
## 🐛 Bug Report

Still tracking down the exact conditions to reproduce this issue, but I first noticed it in the visual states table with Select components placed in table cells:

![image](https://user-images.githubusercontent.com/146020/84327385-0b08da00-ab34-11ea-999c-b59e8b7c966f.png)

Note how the menu is slightly shifted to the right. I've only noticed this bug in IE, Edge, and Firefox (Chrome _appears_ to be unaffected).

This bug was introduced with the use of a portal to render the menu in #641. It likely has something to do with fractional position values of the reference element (the Select button) passed into Popper (the Select menu is positioned using the Canvas Kit Popper which is a thin wrapper around [Popper.js](https://popper.js.org)).

## To Reproduce

Create a table with a width of `100%` and place multiple Selects in different cells within a single row:

```
<table style={{width: '100%'}}>
  <tr>
    <td>
      <Select options={options} />
    </td>
    <td>
      <Select options={options} />
    </td>
    <td>
      <Select options={options} />
    </td>
    <td>
      <Select options={options} />
    </td>
  </tr>
</table>
```

Using IE or Firefox, activate any of the menus and resize the width of the browser window to the point where the Select buttons begin to shift position as the width of the window expands. Note how the menu occasionally loses alignment with its corresponding Select button.

## Expected Behavior

The menu should maintain alignment with the corresponding button.

## Actual Results

See screenshot above.

## Browser (if applicable)

IE11, Edge, and Firefox.

**Comment by jamesfan** [comment]
Since reporting this issue, I've tweaked the way the menu is positioned and the alignment seems to be improved in Firefox and IE (though not perfect).

However, I've also noticed the alignment appears to be dependent on high-DPI vs low-DPI displays. Here's a screenshot using Firefox on my external (non-retina) monitor:

![image](https://user-images.githubusercontent.com/146020/84350034-6efbc480-ab6d-11ea-830d-ddaf2438eeeb.png)

And here's a screenshot of that exact same window after dragging it to my MBP retina screen:

![image](https://user-images.githubusercontent.com/146020/84350133-a5d1da80-ab6d-11ea-8052-c4947110c23b.png)

**Comment by jamesfan** [comment]
This issue was addressed to best of our knowledge in #641. For more information: https://github.com/Workday/canvas-kit/pull/641/commits/7d4e8422dabaf94c3a3d5060b16f3cea7518cc99#r439524681

As it's difficult to test this comprehensively across all scenarios (browsers, high- vs. low-DPI displays, browser window sizes, etc.), I'm marking this issue as closed. We'll re-file it if we see it again in the future.
